src/pages/EditMonitor.vue

```diff
@@ -32,6 +32,9 @@
                                     <option value="steam">
                                         Steam Game Server
                                     </option>
+                                    <option value="docker">
+                                        Docker Container
```

Comment:
Not translated

Reply:
Added the translation, just asking though, why does this get translated, but e.g. "Steam Game Server" doesn't?

Reply:
Some PR authors forget about it and this wasn't catch in Code Review